src/main/java/com/pi4j/crowpi/applications/SoundSensorApp.java

```diff
@@ -0,0 +1,50 @@
+package com.pi4j.crowpi.applications;
+
+import com.pi4j.context.Context;
+import com.pi4j.crowpi.Application;
+import com.pi4j.crowpi.components.SoundSensorComponent;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+/**
+ * A example application to show how the sound sensor could be used. It registers a event handler to the sensors and
+ * the program loops until you clapped 3 times.
+ */
+public class SoundSensorApp implements Application {
+    @Override
+    public void execute(Context pi4j) {
+        System.out.println("Welcome to the SoundSensor Test");
+
+        // Initialize clap counting variable
+        // There is a special kind of integer used because java lambda functions support only final variables
+        AtomicInteger count = new AtomicInteger();
```

Comment:
```suggestion
        final var count = new AtomicInteger();
```

I'd suggest to explicitly make this variable final here as well instead of relying on the "effectively final" clause.